installer/templates/static/brunch/brunch-config.js

```diff
@@ -51,6 +51,13 @@ exports.config = {
       ignore: [/web\/static\/vendor/]
     }
   },
+
+  modules: {
+    autoRequire: {
+      'js/app.js': ['app']
```

Comment:
Changed it and tried it out with the ff brunch_config.js and app.js https://gist.github.com/gjaldon/22190474a0579513fb99. It doesn't seem to load app.js. This is on a new Phoenix app with brunch master. Am I doing it right?